### PR TITLE
Add release of dune-release

### PR DESCRIPTION
**Pull request by Leonidas-from-XIV**
Release in opam-repository: https://github.com/ocaml/opam-repository/pull/23986

### CODE REVIEW

**Comment by tmattio** [comment]
@Leonidas-from-XIV I've added a small paragraph for the announcement. Do you want to have a look and let me know if you'd like to change anything (or make the changes directly if you prefer)

**Comment by Leonidas-from-XIV** [comment]
That's nice, thanks. I added a note on the support in Dune 3.8 because that's sort of convenient and the initial motivation for this addition.

File: data/changelog/dune-release/2023-06-22-dune-release-2.0.0.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    the `opam` binary to be version 2.0 at least. (#406, #411,
+    @Leonidas-from-XIV)
+---
+
+We're excited to announce the release of Dune-release 2.0.0!
+
+This release brings support for putting your `.opam` files in a `opam/`
+directory. If your project contains dozens of packages, you'll be able to
+unclutter your root directory from all of these opam files! You can make Dune
+them into the `opam/` folder starting with Dune 3.8 using `(opam_file_location
```
Comment (tmattio) [inline data/changelog/dune-release/2023-06-22-dune-release-2.0.0.md]:
```suggestion
generate them into the `opam/` folder starting with Dune 3.8 using `(opam_file_location
```

**Comment by Leonidas-from-XIV** [comment]
Updated with the date the project was available on opam-repository.

**Review by sabine — APPROVED** [review]

**Comment by tmattio** [comment]
Thanks a lot @Leonidas-from-XIV and congrats on the release! 🎉

**Pull request merged**